server/src/main/java/org/opensearch/cluster/node/DiscoveryNode.java

```diff
@@ -458,6 +465,14 @@ public boolean isRemoteClusterClient() {
         return roles.contains(DiscoveryNodeRole.REMOTE_CLUSTER_CLIENT_ROLE);
     }
 
+    public String getZoneValue() {
+        return zoneValue;
+    }
+
+    public boolean hasZoneAttribute() {
+        return hasZoneAttribute;
+    }
+
```

Comment:
Can we take a not read the awareness attribute associated with the node at bootstrap and cache it, since it would be fairly static?